### PR TITLE
Using Bootstrap - Milestone 1

### PR DESCRIPTION
**Pull request by verissimohenry**
Using Bootstrap - Milestone 1 to create a Newsweek page

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,17 +1,131 @@
...
+      <a href="#" class="col-lg-1 text-dark">TRENDING</a>
+      <a href="#" class="col-lg-1 text-dark">JOE BIDEN</a>
+      <a href="#" class="col-lg-2 text-dark">DONALD TRUMP</a>
+      <a href="#" class="col-lg-1 text-dark">INAUGURATION</a>
+      <a href="#" class="col-lg-2 text-dark"> &nbsp; &nbsp; &nbsp;CORONAVIRUS</a>
+      <a href="#" class="col-lg-2 text-dark">STIMULUS PACKAGE</a>
+      <a href="#" class="col-lg-1 text-dark">VACCINE</a>
+      <a href="#" class="col-lg-2 text-dark">CAPITOL RIOTS</a>
+      </h6>
+  </div>
```
Comment (BrittanyBlake) [inline index.html]: - [x] This main navigation menu should be using the `<nav/>` tag as you should be using [semantic html](https://www.w3schools.com/html/html5_semantic_elements.asp).
Screenshot from pathwright 👇🏾 
Screen Shot 2021-03-08 at 6 42 27 PM

- [x] You are missing the search form in the main navigation menu
your clone 👇🏾 
Screen Shot 2021-03-08 at 6 59 02 PM

the original page 👇🏾 
Screen Shot 2021-03-08 at 6 59 13 PM

- [x] The spacing and alignment of the elements could be improved.

File: index.html [hunk 2]
```diff
@@ -1,17 +1,131 @@
...
+        </svg>
+      </div>
+      <div class="col-lg-3 col-sm-4 col-md-4 pt-lg-3 mt-lg-4 text-white">
+        <h5>
+          <a href="" class="lg1"><span class="d-none d-lg-block d-md-block d-sm-block">LOGIN</span></a>
+          <a href="" class="lg"><span class="bg-warning text-dark sub">SUBSCRIBE></span><i class="fas fa-bars d-lg-none"></i></a>
+        </h5>
+      </div>
+    </div>
+  </div>
```
Comment (BrittanyBlake) [inline index.html]: ### **GENERAL**
- [x] The NewsWeek logo should be wrapped in an `<a/>` tag.
Screenshot from requirements 👇🏾 
Screen Shot 2021-03-08 at 6 45 12 PM

### **DESKTOP**
- [x] The logo should also be centered on the page, it is not centered on my browser.
Your clone 👇🏾 
Screen Shot 2021-03-08 at 6 46 17 PM
The original page 👇🏾 
Screen Shot 2021-03-08 at 6 46 57 PM

### **TABLET & MOBILE**

- [x] The elements aren't properly aligned on the tablet and mobile screen sizes.
Your clone on Tablet 👇🏾 
Screen Shot 2021-03-08 at 6 49 08 PM

Your clone on Mobile 👇🏾 
Screen Shot 2021-03-08 at 6 50 38 PM

The original page on Tablet 👇🏾 
Screen Shot 2021-03-08 at 6 49 25 PM

The original page on Mobile 👇🏾 
Screen Shot 2021-03-08 at 6 50 59 PM

File: index.html [hunk 3]
```diff
@@ -1,17 +1,131 @@
...
-  <title>Document</title>
+  <title>NewsWeek Clone</title>
 </head>
-
 <body>
-  <h1>hello world</h1>
+  <!--HEADER-->
+  <div class="container-fluid first d-flex flex-sm-row flex-md-row">
+    <div class="row">
+      <div class="col-lg-3 pt-lg-3 mt-lg-4 text-white d-none d-lg-block">Wed, Jan 20, 2021</div>
```
Comment (BrittanyBlake) [inline index.html]: - [x] The date should not wrap to the other line.
Your clone 👇🏾 
Screen Shot 2021-03-08 at 6 54 26 PM
The original page 👇🏾 
Screen Shot 2021-03-08 at 6 54 40 PM

File: index.html [hunk 4]
```diff
@@ -1,17 +1,131 @@
...
+        </svg>
+      </div>
+      <div class="col-lg-3 col-sm-4 col-md-4 pt-lg-3 mt-lg-4 text-white">
+        <h5>
+          <a href="" class="lg1"><span class="d-none d-lg-block d-md-block d-sm-block">LOGIN</span></a>
```
Comment (BrittanyBlake) [inline index.html]: - [x] The login button should change to an icon on mobile screens.
Your clone 👇🏾 
Screen Shot 2021-03-08 at 6 50 38 PM

The original page👇🏾 
Screen Shot 2021-03-08 at 6 50 59 PM

File: style.css [hunk 5]
```diff
@@ -1,8 +1,27 @@
...
+}
+
+.first {
+  background-color: #f72210;
+}
+
+a {
+  text-decoration: none;
+  color: white;
+}
```
Comment (BrittanyBlake) [inline style.css]: - [x] This can be done with the bootstrap classes `text-decoration-none` and `text-white`

File: style.css [hunk 6]
```diff
@@ -23,17 +42,167 @@ ul {
...
+
+.first-bottom {
+  background-color: #f72210;
+  border-bottom: 2px solid #eee;
+  width: 66px !important;
+  position: absolute;
+  top: 88px;
+  right: 0;
+  content: "";
+  display: block;
```
Comment (BrittanyBlake) [inline style.css:121]: - [x] This can be done with the bootstrap class `d-block`

File: style.css [hunk 7]
```diff
@@ -23,17 +42,167 @@ ul {
...
+  border-left: 12px solid white;
+  border-top: 13px solid #f72210;
+}
+
+.content {
+  position: fixed;
+  bottom: 0;
+}
```
Comment (BrittanyBlake) [inline style.css]: - [x] You can make your footer stick to the bottom by adding the following bootstrap class `fixed-bottom`

**Review by BrittanyBlake — CHANGES_REQUESTED** [review]
### Status: Changes Required ♻️ 
****
### Hi @verissimohenry and @tongoonamujera! 👋 

You have done an amazing job with this milestone! It looks really good 👏 👏 👏  There are just a few issues that need to be resolved before you can move on: ⬇️ 
****

- [x] Check my comments for more changes.

- [x] Some of your css styles can be done with bootstrap classes. Take a look at your css file again and use this [cheatsheet](https://hackerthemes.com/bootstrap-cheatsheet/) to see what can be done with bootstrap instead. 
****
Good Luck and Happy Coding 🍀 👩🏾‍💻

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Brittany Blake)._

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
# Status: Changes Required :vertical_traffic_light: :warning: 
## Hello @verissimohenry and @tongoonamujera  :handshake: :handshake: :handshake: 

:+1: This is good to make it this far, your clone looks like the original page and it is really nice, congratulations on the good work you have done so far.
:+1: I liked that you have tried to implemented the requested changes from the previous review, keep it up, and your project is really almost done.
#### :warning: However we need to make few changes to mark this project approved, don't feel discouraged because of being asked some changes more and more, that's beacuse this is a learning process, and we are here to help you to become an out-standing developer. See the requested change below :point_down: :point_down: :point_down: 
### Note: All of requested changes which are marked as [Required] are mandatory to be implemented in order to get this project approved, others which are marked as [Optional] are nice to have in your project to make it outstanding

# [Required Changes] :red_circle: 
- [x] Remove the `.vscode` folder from your project. It's not related to the codes.
- [x]  You are still missing the search form in the main navigation menu
your clone => You have added a search icon, but that doesn't mean a search form. [Refer to this resource](https://www.w3schools.com/html/html_forms.asp) to know how to add html form
- [x] The NewsWeek logo should be wrapped in an <a/> tag. => It was required in the previous review
- [x] The elements aren't properly aligned on the tablet and mobile screen sizes.

=> Footer on mobile screens (width 320px)

| Your clone | Original page |
| --- | --- |
| Screenshot from 2021-03-09 14-00-46 | Screenshot from 2021-03-09 14-00-37 |\

:point_right: Could you see the difference :question: :question: :question: 

=> Navbar on mobile screens (width 320px)

| Your clone | Original page |
| --- | --- |
| Screenshot from 2021-03-09 14-00-18 | Screenshot from 2021-03-09 14-00-06|

:point_right: Could you see the difference :question: :question: :question: 

- [x] As this project's main focus is the usage of bootstrap, you should focus on using Bootstrap utility classes instead of writing your own css styles. Currently you have al lot of css properties which could be removed and replaced by bootstrap classes. For example the `display: none;` can be replaced by `.d-none`, `color: white;` can be replaced by `.text-white` :point_left: Kindly revise your css file, and refer to [this resource](https://getbootstrap.com/docs/5.0/getting-started/introduction/) and [this one](https://www.w3schools.com/bootstrap4/bootstrap_utilities.asp) to refactor all of the styles which have alternatives in bootstrap.
___
_If anything seems to be unclear to you, reachout and ask anything by commenting on this Pull Request and mention me as [descholar-ceo](https://github.com/descholar-ceo), I will receive a notification and will respond._

[GitHub](https://github.com/descholar-ceo) | [LinkedIn](https://linkedin.com/in/mugirase-emmanuel) | [Twitter](https://twitter.com/descholar3)

_Reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)_

**Review by mahmoud717 — CHANGES_REQUESTED** [review]
### Changes required ♻️ 

**Hello @verissimohenr**

Great job with the project so far it looks amazing, but there are still some changes to be made:
 
---
### Requested 🔴 

- [x] you should settle on one way for positioning your elements. for instance, in the navbar you used flexbox and relative positioning. you should just stick with flexbox. in this case, using flexbox with justify-content and align-items properties should simply do the trick. and it would also make it a lot easier to make the navbar responsive.
- [x]  the font-size in the sun-navbar is too small, make sure to enlarge it.

---

please make these changes and request another code review.

if you have any questions don't hesitate to contact me on slack **MahmoudMohammad**

**Review by lekegitrepo — APPROVED** [review]
## Status: Approved :heavy_check_mark:
### Good job Team :+1:
#### This project is approved :white_check_mark:

#### _Reviewed by Temitayo Adeleke_

**Pull request merged**